nmostesting/ControllerTest.py

```diff
@@ -79,6 +79,11 @@ class ControllerTest(GenericTest):
     def __init__(self, apis, registries, node, dns_server, disable_auto=True):
         # Remove the spec_path as there are no corresponding GitHib repos for Controller Tests
         apis[CONTROLLER_TEST_API_KEY].pop("spec_path", None)
+        if apis[CONTROLLER_TEST_API_KEY]["base_url"].startswith("https"):
```

Comment:
Could/should this just check `ENABLE_HTTPS` ?
Or check both `"base_url"` and `"url"` separately?

Reply:
Yes should check `ENABLE_HTTPS` I think.  Admittedly attempting to substitute `https` for `http` will just not nothing in the `http` case, but I think the condition makes the intention explicit.